ui/admin/setup-edit.php

```diff
@@ -168,7 +168,7 @@
                     ),
                     'class' => 'pods-validate pods-validate-required'
                 ) ); ?>
-                    <input type="button" class="save-button button" value="OK" /> <a class="cancel" href="#cancel-edit">Cancel</a>
+                    <input type="button" class="save-button button" value="<?php _e( 'OK', 'pods' ); ?>" /> <a class="cancel" href="#cancel-edit"><?php _e( 'Cancel', 'pods' ); ?></a>
```

Comment:
Please use `esc_attr_e()` instead of `_e()` for echoing values in an attribute.

Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.